XRTK-Core/Packages/com.xrtk.core/Providers/Controllers/UnityInput/GenericJoystickController.cs

```diff
@@ -85,10 +85,24 @@ protected void UpdateButtonData(MixedRealityInteractionMapping interactionMappin
         {
             Debug.Assert(interactionMapping.AxisType == AxisType.Digital);
 
-            var keyButton = Input.GetKey(interactionMapping.KeyCode);
-
             // Update the interaction data source
-            interactionMapping.BoolData = keyButton;
+            switch (interactionMapping.InputType)
+            {
+                case DeviceInputType.TriggerPress:
```

Comment:
So the now removed: `case DeviceInputType.ButtonPress` was wrong? This should now handled in the `default:` ?

Reply:
Yes, button press and and other digital inputs that are not axis are handled in the default

But this does pose another case. This is based on unitys openvr spec. Maybe we need an additional bit of meta data about the input to identify source axis and target axis to better identify how to transform the data. Doing it based on the input type is potentially too limiting.

Will stick to how this is now as this solves the immediate issue. 

But there has been an additional comment in the MRTK space I want to review before merging, about getaxis and getrawaxis